geoportal/c2cgeoportal_geoportal/scaffolds/update/geoportal/CONST_vars.yaml_tmpl

```diff
@@ -871,6 +873,14 @@ vars:
         Content-Security-Policy: 'default-src {content_security_policy_c2c_default_src};
           script-src {content_security_policy_c2c_script_src};
           style-src {content_security_policy_c2c_style_src};'
+    - pattern: '^/iframe_api?.*'
```

Comment:
```suggestion
    - pattern: '^/iframe_api([/\?].*)?$'
```

Reply:
Fixed.
In fact it works perfectly with '^/iframe_api$' as it is matched with request.path_info which only contains path and not query string.

Reply:
But the theme is missing, or also not present?

Reply:
my bad, forgot about theme in path, maybe it would me more restrictive and readable with : `"^/iframe_api(/theme/.*)?$"` which match `"/iframe"` and `"/iframe_api/theme/Demo"` for example.

Reply:
Looks good :-)